src/recommendations/events.md

```diff
@@ -7,16 +7,16 @@ When you deploy the `product-recommendations` module, the module tracks the foll
 
 Event |Description
 --- | ---
-`add-to-cart` | Triggered when the shopper adds an item to their cart
-`add-to-wishlist` | Triggered when the shopper adds an item to their wishlist
-`apply-discount-action` | Triggered when the shopper applies a discount to their cart
-`edit-product-qty` | Triggered when the shopper modifies the number of items in their cart
+`add-to-cart` | Triggered when a cart updates with a new item
+`add-to-wishlist` | Triggered when a wishlist updates with a new item
+`apply-discount-action` | Triggered when a discount is applied to a cart
```

Comment:
See! this one is passive voice already
I would amend to "discount code is applied"